src/custom/components/swap/EthFlow/pure/EthFlowModalContent/configs.ts

```diff
@@ -0,0 +1,119 @@
+    nativeSymbol: string
+    wrappedSymbol: string
+  }) => EthFlowConfig
+} = {
+  /**
+   * FAILED operations
+   * wrap/approve/both in expertMode failed
+   */
+  [EthFlowState.WrapAndApproveFailed]: () => ({
+    title: 'Wrap and Approve failed!',
+    buttonText: 'Wrap and approve',
+    descriptions: [
+      'Both wrap and approve operations failed.',
+      `Check that you are providing a sufficient gas limit for both transactions in your wallet. Click "Wrap and approve" to try again`,
+    ],
+  }),
+  [EthFlowState.WrapFailed]: ({ nativeSymbol }) => ({
+    title: `Wrap ${nativeSymbol} failed`,
+    buttonText: `Wrap ${nativeSymbol}`,
+    descriptions: [
+      `Wrap operation failed.`,
+      `Check that you are providing a sufficient gas limit for the transaction in your wallet. Click "Wrap ${nativeSymbol}" to try again`,
+    ],
+  }),
+  [EthFlowState.ApproveFailed]: ({ wrappedSymbol }) => ({
+    title: `Approve ${wrappedSymbol} failed!`,
+    buttonText: `Approve ${wrappedSymbol}`,
+    descriptions: [
+      `Approve operation failed. Check that you are providing a sufficient gas limit for the transaction in your wallet`,
+      `Click "Approve ${wrappedSymbol}" to try again`,
+    ],
+  }),
+  /**
+   * PENDING operations
+   * wrap/approve/both in expertMode
+   */
+  [EthFlowState.WrapAndApprovePending]: () => ({
+    title: 'Wrap and approve',
+    buttonText: '',
+    descriptions: ['Transactions in progress', 'See below for live status updates of each operation'],
+  }),
+  [EthFlowState.WrapPending]: ({ nativeSymbol }) => ({
+    title: `Swap with Wrapped ${nativeSymbol}`,
+    buttonText: '',
+    descriptions: ['Transaction in progress. See below for live status updates'],
+  }),
+  [EthFlowState.ApprovePending]: ({ wrappedSymbol }) => ({
+    title: `Approve ${wrappedSymbol}`,
+    buttonText: '',
+    descriptions: ['Transaction in progress. See below for live status updates'],
+  }),
+  [EthFlowState.ApproveInsufficient]: ({ wrappedSymbol }) => ({
+    title: 'Approval amount insufficient!',
+    buttonText: `Approve ${wrappedSymbol}`,
+    descriptions: [
+      'Approval amount insufficient for input amount',
+      'Check that you are approving an amount equal to or greater than the input amount',
+    ],
+  }),
+  /**
+   * NEEDS operations
+   * need to wrap/approve/both in expertMode
+   */
+  [EthFlowState.WrapAndApproveNeeded]: ({ nativeSymbol }) => ({
+    title: 'Wrap and approve',
+    buttonText: '',
+    descriptions: [
+      `2 pending on-chain transactions: ${nativeSymbol} wrap and approve. Please check your connected wallet for both signature requests`,
+    ],
+  }),
+  [EthFlowState.WrapNeeded]: ({ isExpertMode, nativeSymbol }) => ({
+    title: `Swap with Wrapped ${nativeSymbol}`,
+    buttonText: `Wrap ${nativeSymbol}`,
+    descriptions: isExpertMode
+      ? [expertCommonDescription]
+      : [ethFlowDescription(nativeSymbol), 'TODO: You dont have enough wrapped token...'],
```

Comment:
Maybe you wanna update it already here?

Reply:
Thank you!
I'm not sure about the text, I hope @anxolin will fix it